src/Helper/Placeholder/Container/AbstractStandalone.php

```diff
@@ -176,6 +176,24 @@ protected function escape($string)
         return $this->getEscaper()->escapeHtml((string) $string);
     }
 
+    /**
+     * Escape an attribute value
+     *
+     * @param  string $string
+     * @return string
+     */
+    protected function escapeAttribute($string)
+    {
+        if ($this->getView() instanceof RendererInterface
+            && method_exists($this->getView(), 'getEncoding')
```

Comment:
What's going on here, exactly?

Reply:
@froschdesign Exactly. I think we could safely drop the if statement and use the return statement only.